compiler/damlc/lib/DA/Cli/Visual.hs

```diff
@@ -26,8 +26,8 @@ data Action = ACreate (LF.Qualified LF.TypeConName)
             | AExercise (LF.Qualified LF.TypeConName) LF.ChoiceName deriving (Eq, Ord, Show )
 
 data ChoiceAndAction = ChoiceAndAction
-    { choiceForTemplate :: LF.Template
-    , choice :: LF.TemplateChoice
+    { choiceName :: LF.ChoiceName
+    , consumingChc :: IsConsuming
```

Comment:
I would get rid of `Chc` or maybe use `choiceConsuming`.

Reply:
cool